src/app/main/error/error.component.ts

```diff
@@ -8,14 +17,90 @@ import { Title } from '@angular/platform-browser';
 })
 export class ErrorComponent implements OnInit {
 
-    @Input() status?: number = 404;
+    @Input() status: number;
+
+    // default error messages
+    errorMessages: ErrorMsg[] = [
+        {
+            status: 0,
+            message: "Undefined Error",
+            description: `Maybe I'm a teapot, maybe not.<br>
+            But anyway, something undefined went wrong.`,
+            action: 'goback',
+            image: 'dsp-error.svg'
+        },
+        {
+            status: 403,
+            message: "Forbidden",
+            description: `This is not the content you were looking for.<br>
```

Comment:
I would say "Invalid permissions." instead of "This is not the content you were looking for." because the user was likely looking for this content but just doesn't have the permissions.

Reply:
Done in 686f1ce